content/go/notification_ocurrent_events.go.md

```diff
@@ -0,0 +1,235 @@
+---
+title: "Managing Notification Occurrence Events services"
+description: "How to interact with the Notification Ocurrence Events services. The SoftLayer_Notification_Occurrence_Event service represents all events with potential to cause a disruption in service.."
+date: "2020-10-07"
+classes: 
+    - "SoftLayer_Virtual_Guest"
+    - "SoftLayer_Metric_Tracking_Object"
+tags:
+    - "Metric Tracking Object "    
+---
+
+# Setup
+Each of these snippets below will share basically the same initialization code, so to save some space we will include the initialization code here, and assume you can setup the SoftLayer client before running each example.
+
+```Go
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/softlayer/softlayer-go/services"
+	"github.com/softlayer/softlayer-go/session"
+	"github.com/softlayer/softlayer-go/sl"
+)
+
+func main() {
+	username := "Set - me"
+	apikey := "Set - me"
+
+	// Create SoftLayer API session
+	sess := session.New(username, apikey)
+
+	notificationService := services.GetNotificationOccurrenceEventService(sess)
+
+	fmt.Println(getAllObjects(notificationService))
+	// Notificacion ocurrent event id = 123456
+    fmt.Println(getObject(notificationService,123456))
+}
+
+func getAcknowledgeNotification(service services.Notification_Occurrence_Event, notificationId int) bool {
+	service.Options.Id = sl.Int(notificationId)
+	result, err := service.AcknowledgeNotification()
+	if err != nil {
+		fmt.Printf("\n Unable to place order:\n - %s\n", err)
+
+	}
+	return result
+}
+
+func getAllObjects(service services.Notification_Occurrence_Event) string {
+	receipt, err := service.GetAllObjects()
+	if err != nil {
+		fmt.Printf("\n Unable to place order:\n - %s\n", err)
+	}
+
+	jsonFormat1, jsonErr1 := json.MarshalIndent(receipt, "", "     ")
+	if jsonErr1 != nil {
+		fmt.Println(jsonErr1)
+	}
+	return (string(jsonFormat1))
+}
+
+func getAttachments(service services.Notification_Occurrence_Event, notificationId int) string {
+	service.Options.Id = sl.Int(notificationId)
+	receipt, err := service.GetAttachments()
+	if err != nil {
+		fmt.Printf("\n Unable to place order:\n - %s\n", err)
+	}
+
+	jsonFormat1, jsonErr1 := json.MarshalIndent(receipt, "", "     ")
+	if jsonErr1 != nil {
+		fmt.Println(jsonErr1)
+	}
+	return (string(jsonFormat1))
+}
+
+func getFirstUpdate(service services.Notification_Occurrence_Event, notificationId int) string {
+	service.Options.Id = sl.Int(notificationId)
+	receipt, err := service.GetFirstUpdate()
```

Comment:
You can just do this for setting the Id

```go
result, err := service.Id(notificationId).GetFirstUpdate()
```
Change all these to follow that format as that is more the "golang" way of using the library.